config/config.go

```diff
@@ -0,0 +1,14 @@
+package config
+
+type Config struct {
+	Environment   *string
+	MongoUri      *string
+	MongoDbName   *string
+	JwtPrivateKey *string
+}
+
+var (
+	EnvIsProd = "production"
+	EnvIsDev  = "development"
+	Conf      *Config
+)
```

Comment:
I don't think it's necessary to have a `config` package for a couple variables. I don't think they clutter the main function that much. Up to you!